src/style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  --moyo-blue: #00acdc;
+}
+
+.picture-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.picture-container img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain;
+}
+
+.product-card p {
+  margin: 0;
+  user-select: none;
```

Comment:
why you add it?

Reply:
to nobody select data (deleted already)

Reply:
why do you need such behaviour ?

Reply:
i think it can be useful for user. To dont select text when accidentally touch.
(but I already delete it)